dockerfiles/Picard/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM alpine:latest
+
+ARG version=2.22.3
+
+# make sure the package repository is up to date
+RUN apk update \
+ && apk upgrade \
+ && apk add bash \
+ && rm -rf /var/cache/*/* \
+ && echo "" > /root/.ash_history
+
+# change default shell from ash to bash
```

Comment:
can be achieved by using image `bashell/alpine-bash:latest` as below?

Reply:
bashell/alpine-bash:latest hasn't been updated for a while. I did want to use a later version of a package while making this dockerfile but it looks like I didn't use it in the end. I will revert it back to alpine-bash.